wordpress-scripts-webpack.config.js

```diff
@@ -1,6 +1,7 @@
 /**
  * External Dependencies
  */
+const fs = require('fs');
```

Comment:
We don't need this anymore

Reply:
Fixed on 21570442e5270389a139362a565d60dbee5c1149